lib/resources.js

```diff
@@ -657,9 +675,15 @@ module.exports = function (app, _private = null) {
    * @returns {object}
    */
   const itemsQueryContext = (options) => {
-    return options.merge_checkin_card_items
-      ? { must: { match_all: {} } }
-      : { must_not: { term: { 'items.type': 'nypl:CheckinCardItem' } } }
+    if (options.merge_checkin_card_items && options.removeElectronicResourcesFromItemsArray) {
+      return { must_not: { exists: { field: 'items.electronicLocator' } } }
+    }
+
+    if (options.merge_checkin_card_items) {
```

Comment:
i dont believe this is necessary. im pretty sure there are not electronic locators for check in card items